test/integration/functional_test.go

```diff
@@ -324,21 +324,24 @@ func listImages(ctx context.Context, t *testing.T, profile string) (*RunResult,
 
 // validateBuildImage makes sures that `minikube image build` works as expected
 func validateBuildImage(ctx context.Context, t *testing.T, profile string) {
+	// docs(skip): Skips on non driver since load image is not available on none driver
 	if NoneDriver() {
 		t.Skip("load image not available on none driver")
```

Comment:
It would be cool if we can just parse out the skip info from the parameter of `t.Skip`. However, it's not doable for `t.Skipf` calls because we can't get the evaluated string without actually running the code, e.g.:

```go
t.Skipf("only validate docker env with docker container runtime, currently testing %s", cr)
```

So I added a `docs(skip): ` tag to document the skip cases.

An alternative approach is that we make sure we are only using `t.Skip`:

```diff
- t.Skipf("only validate docker env with docker container runtime, currently testing %s", cr)
+ t.Skip("only validate docker env with docker container runtime")
```

Then it's possible to parse out the string from the function parameter.